pkg/apis/serving/v1alpha1/route_types.go

```diff
@@ -107,9 +109,13 @@ const (
 	// service is not configured properly or has no available
 	// backends ready to receive traffic.
 	RouteConditionAllTrafficAssigned duckv1alpha1.ConditionType = "AllTrafficAssigned"
+
+	// RouteConditionIngressReady is set to False when the
+	// ClusterIngress fails to become Ready.
+	RouteConditionIngressReady duckv1alpha1.ConditionType = "IngressReady"
```

Comment:
Ah, thanks for reminding.
So the doc is automatically updated (through tools/scripts), or updated manually?

Reply:
Manual.  If you know of good tooling, let us know :)

Reply:
Still I did not update the doc. Because as it indicates [here](https://github.com/knative/serving/blob/master/pkg/reconciler/v1alpha1/clusteringress/clusteringress.go#L152), currently we can only see `Ready` and `Unknown` in ingress status, and no known errors yet. 
I'd like to revisit this when we have way to probe virtualservice status.

Reply:
Let me bring this up in WG meet up today.  Yes, I'd love to see more correctness in Route.Status.Ready.